.circleci/config.yml

```diff
@@ -52,10 +52,10 @@ jobs:
     <<: *defaults
     parameters:
       docker-compose-filename:
-        default: docker-compose.circleci.yml
+        default: docker-compose.yml
         type: string
       docker-image-name-backend:
-        default: prisoner-content-hub-backend
+        default: drupal
```

Comment:
No need to prefix the image name anymore, as docker-compose will do that for us with the project name.  This means we get a docker name: `prisoner-content-hub-backend-drupal-1` instead of `prisoner-content-hub-backend-prisoner-content-hub-backend-1`